src/com/xilinx/rapidwright/router/VersalClockRouting.java

```diff
@@ -261,18 +347,16 @@ public static Map<ClockRegion, Node> routeVerticalToHorizontalDistributionLines(
             
             while (!q.isEmpty()) {
                 NodeWithPrev curr = q.poll();
+                if (getNodeStatus.apply(curr) != NodeStatus.AVAILABLE) {
+                    continue;
+                }
                 NodeWithPrev parent = curr.getPrev();
                 if (targetCR.equals(curr.getTile().getClockRegion()) &&
                     curr.getIntentCode() == IntentCode.NODE_GLOBAL_GCLK &&
                     parent.getIntentCode() == IntentCode.NODE_GLOBAL_HDISTR_LOCAL) {
                     List<Node> path = curr.getPrevPath();
                     for (int i = 1; i < path.size(); i++) {
                         Node node = path.get(i);
-                        NodeStatus status = getNodeStatus.apply(node);
-                        if (status == NodeStatus.INUSE) {
-                            break;
-                        }
```

Comment:
I have a inkling that this code was for incremental clock routing. However -- assuming we have some -- incremental tests still seem to be passing. Maybe we need to run the ECO flow.